dev/fips202/aarch64/src/keccak_f1600_x1_scalar_asm.S

```diff
@@ -8,6 +8,31 @@
 // Author: Hanno Becker <hanno.becker@arm.com>
 // Author: Matthias Kannwischer <matthias@kannwischer.eu>
 
+/*
+ * Function: keccak_f1600_x1_scalar_asm
+ * Description: AArch64 scalar implementation of Keccak-f[1600] permutation for single state
+ *
+ * C Signature: void mlk_keccak_f1600_x1_scalar_asm(uint64_t state[25], const uint64_t rc[24])
+ *
+ * Register ABI:
+ * ```yaml
+ *   x0:
+ *     type: buffer
+ *     size_bytes: 200
+ *     permissions: read/write
+ *     c_parameter: uint64_t *state
```

Comment:
```suggestion
 *     c_parameter: uint64_t state[25]
```
Same for other arguments in the fips202 backend.

Reply:
Done